hawc/apps/animal/forms.py

```diff
@@ -144,23 +144,23 @@ def __init__(self, *args, **kwargs):
             self.instance.experiment = parent
 
         # for lifestage assessed/exposed, use a select widget. Manually add in
-        # previously saved values that don't conform to the LIFESTAGE_CHOICES tuple
-        lifestage_dict = dict(models.AnimalGroup.LIFESTAGE_CHOICES)
+        # previously saved values that don't conform to the lifestage choices
+        lifestage_dict = dict(constants.Lifestage.choices)
 
         if self.instance.lifestage_exposed in lifestage_dict:
-            le_choices = models.AnimalGroup.LIFESTAGE_CHOICES
+            le_choices = constants.Lifestage.choices
         else:
             le_choices = (
                 (self.instance.lifestage_exposed, self.instance.lifestage_exposed),
-            ) + models.AnimalGroup.LIFESTAGE_CHOICES
+            ) + constants.Lifestage.choices
```

Comment:
does this actually work? it looks like it should, but wanted to make sure...

Reply:
Yep!
![image](https://user-images.githubusercontent.com/46504563/144447002-5ce9ca7e-7896-48be-91f7-0601ebc447eb.png)